grammar.js

```diff
@@ -65,6 +66,7 @@ module.exports = grammar({
     [$.scoped_identifier, $.scoped_type_identifier],
     [$.parameters, $._pattern],
     [$.parameters, $.tuple_struct_pattern],
+    [$.await_expression, $.field_expression],
```

Comment:
I would not expect an LR(1) conflict to arise because of await expressions. What conflict error message do you get if you remove this?

Reply:
None, I'll remove on the next commit